x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
 {
   "template": {
     "mappings": {
+      "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "runtime": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "message": {
+            "path_match": "message",
```

Comment:
Lets only index `message` on the top level. There is a good chance that also for example `error.message` exists but I don't think we should index it by default.

Reply:
Hm, I might mix `match` and `patch_match`?

Reply:
Yes, `"path_match": "message"` only indexes `message` and doesn't index `error.message`. If I used `"match": "message"` or "path_match": "*message", `error.message` would be indexed, too.